theano/gpuarray/conv_desc.c

```diff
@@ -1,12 +1,16 @@
 #section support_code_apply
 
-int APPLY_SPECIFIC(conv_desc)(PyArrayObject *filt_shp,
+int APPLY_SPECIFIC(conv_desc)(PyArrayObject *filt_shp, PyArrayObject *subsample,
```

Comment:
Here you also need to add border_mode, mostly, all parameter in make_node must also be here, plus the returned value (desc).